tests/jenkins/TestDataPrepperReleaseArtifacts.groovy

```diff
@@ -61,20 +62,33 @@ class TestDataPrepperReleaseArtifacts extends BuildPipelineTest {
     void 'downloads from the correct URLs'() {
         runScript('jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile')
 
-        def curlCommands = helper.callStack.findAll { call ->
+        def shCommands = helper.callStack.findAll { call ->
             call.methodName == 'sh'
         }.collect { call ->
             callArgsToString(call)
-        }.findAll { shCommand ->
```

Comment:
You should be able to chain `findAll`. So you can still get the `sh` and then filter by `curl`. Say somebody later adds something like the following:

```
mkdir tar
```

That command would end up in the `archiveCommands` array and cause `assertThat(archiveCommands.size(), equalTo(2))` to fail.